python/sglang/srt/layers/moe/fused_moe_triton/layer.py

```diff
@@ -205,19 +224,45 @@ def forward_cpu(
         correction_bias: Optional[torch.Tensor] = None,
         activation: str = "silu",
     ) -> torch.Tensor:
-        return moe_forward_native(
-            layer,
-            x,
-            use_grouped_topk,
-            top_k,
-            router_logits,
-            renormalize,
-            topk_group,
-            num_expert_group,
-            custom_routing_function,
-            correction_bias,
-            activation,
-        )
+        assert activation == "silu", f"activation = {activation} is not supported."
+
+        if support_amx():
```

Comment:
replace with cpu_has_amx_support, again here we need to make sure 1) cpu has amx; 2) sglang has cpu csrc files compiled.

```
if cpu_has_amx_support():
    from sgl_kernel.ops._kernels import fused_experts_cpu

    select_experts(...)

    fused_experts_cpu(...)

```

Reply:
Fixed in https://github.com/mingfeima/sglang/pull/8/commits/533027853f5681629db97387d1d693d653808bbf